migrations/Version30000_4.php

```
@@ -0,0 +1,130 @@
+        $table->addForeignKeyConstraint(
+            'companies',
+            ['company_id'],
+            ['id'],
+            ['onDelete' => 'CASCADE'],
+        );
+    }
+
+    /**
+     * @throws Exception
+     */
+    public function postUp(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)